### ISSUE TITLE
Training loss/Acc diagram

### ISSUE THREAD

**Issue by LisaBernhardt** [SysCV/qdtrack#36]
Thanks for the great work!

I am trying to retrain QDTrack on BDD100k, however, it is converging really slowly (at least for the first epochs). Therefore I wanted to ask, whether it is possible to share your diagrams on training loss and acc?

Thanks in advance!

**Comment by tobiasfshr** [comment]
Hi, unfortunately we have no tensorboard log available but we can provide you with a log file. From this loss diagrams can be calculated / you can look up specific values that you're interested in at a specific epoch / step.
[log_file.zip](https://github.com/SysCV/qdtrack/files/6433429/log_file.zip)

**Comment by LisaBernhardt** [comment]
Thank you for your help!

**Comment by hE8UrdZQ9K6s6pWC** [comment]
@tobiasfshr thank you for the log file. Do you know what GPU configuration the machine in the log was run in? I'm sitting on an 8GB RTX 2070 and am also hoping to retrain the track detector. Perhaps it is not enough power, since my ETA is 20 days. I'm guessing pretrained weights are not available for this project yet

**Comment by LisaBernhardt** [comment]
@hE8UrdZQ9K6s6pWC I think 20 days is a bit long. For reference, it took me 9 days to retrain and reproduce the numbers of the authors with 2 GeForce GTX 1080 Ti.

What looks a bit different here is the batch size (I assume it is?). You load 171715 images per Iteration `Epoch [1][50/**171715**]`. For me it was 68500 and in the log file of the authors it is only 21465.

@tobiasfshr I was searching the config file as well as the code, but I can't see where I can change the batch size. I would really appreciate if you could give me a hint here. Additionally, could you please explain how it's possible that we have different numbers here. I load 68500 per iteration and in the logfile you have send its only 21465.

Thank you for your help!

Example line of my logfile: 
`2021-05-02 04:32:57,590 - qdtrack - INFO - Epoch [4][3900/68500]	lr: 4.000e-02, eta: 5 days, 3:25:14, time: 0.677, data_time: 0.006, memory: 5881, loss_rpn_cls: 0.0396, loss_rpn_bbox: 0.0337, loss_cls: 0.2359, acc: 91.3760, loss_bbox: 0.1339, loss_track: 0.0997, loss_track_aux: 0.0287, loss: 0.5715`

**Comment by tobiasfshr** [comment]
Hi, the log file trained using 8 RTX 2080 Ti GPUs. You can set the batch size per GPU here: https://github.com/SysCV/qdtrack/blob/8fb59ad12652bfdad8818f3e4cb4546fc71f8b4e/configs/qdtrack-frcnn_r50_fpn_12e_bdd100k.py#L199
If you want to reproduce our results on a different number of GPUs / images per GPU, make sure to apply the linear learning rate scaling rule: We train on 8 GPUs using batch size of 2 per GPU, i.e. global BS=16. The learning rate is 0.02 in this setting. If you train on a single GPU using e.g. 4 images per GPU, you need to set the LR to 0.02 * 4 / 16 = 0.005

**Comment by tobiasfshr** [comment]
@hE8UrdZQ9K6s6pWC given the model trained 2.5 days on 8 GPUs, training for 2.5 * 8 = 20 days on one GPU is exactly the amount of time i would expect. You can find all pretrained models under "Usages" in README.md.

**Comment by LisaBernhardt** [comment]
Thank you very much! This really helped :)
